examples/nlp/language_modeling/constants.py

```diff
@@ -132,4 +133,192 @@
         'subset': 'SQuALITY/validation.jsonl',
         'chat_instruction': '',
     },
+    "narrativeqa_lb": {
+        "context": "Story:",
+        "query_start": "Question:",
+        "response": "Answer:",
+        "tokens_to_generate": 128,
+        "subset": "LongBench/narrativeqa.jsonl",
+        "chat_instruction": "",
+        "template": "You are given a story, which can be either a novel or a movie script, and a question. Answer the question asconcisely as you can, using a single phrase if possible. Do not provide any explanation.\n\nStory: {context}\n\nNow, answer the question based on the story asconcisely as you can, using a single phrase if possible. Do not provide any explanation.\n\nQuestion: {input}\n\nAnswer:",
+    },
+    "qasper_lb": {
+        "context": "Article:",
+        "query_start": "Question:",
+        "response": "Answer:",
+        "tokens_to_generate": 128,
+        "subset": "LongBench/qasper.jsonl",
+        "chat_instruction": "",
+        "template": "You are given a scientific article and a question. Answer the question as concisely as you can, using a single phrase or sentence if possible. If the question cannot be answered based on the information in the article, write \"unanswerable\". If the question is a yes/no question, answer \"yes\", \"no\", or \"unanswerable\". Do not provide any explanation.\n\nArticle: {context}\n\n Answer the question based on the above article as concisely as you can, using a single phrase or sentence if possible. If the question cannot be answered based on the information in the article, write \"unanswerable\". If the question is a yes/no question, answer \"yes\", \"no\", or \"unanswerable\". Do not provide any explanation.\n\nQuestion: {input}\n\nAnswer:",
```

Comment:
Instead of adding `\"` in the string, should we replace with a single quote: `'`. Would make it a bit cleaner imo.